src/Microsoft.Extensions.WebEncoders/Testing/HtmlTestEncoder.cs

```diff
@@ -18,6 +18,11 @@ public override int MaxOutputCharactersPerInputCharacter
 
         public override string Encode(string value)
         {
+            if (string.IsNullOrEmpty(value))
```

Comment:
Careful, you've moved this behavior from production code to test code and you're now mis-representing the actual encoder's behaviors for null. https://github.com/dotnet/corefx/blob/c02d33b18398199f6acc17d375dab154e9a1df66/src/System.Text.Encodings.Web/src/System/Text/Encodings/Web/TextEncoder.cs#L44